example-cropping/src/components/SignUp.jsx

```diff
@@ -44,7 +45,8 @@ export default function SignUp({ currentPage, appwrite, setCurrentPage }) {
       return;
     }
     try {
-      await appwrite.account.create(email, password);
+      const account = new Account(appwrite)
+      await account.create("unique()",email, password);
```

Comment:
You can now import `ID` from `appwrite` for helpers like unique.
Then update like below
```
      await account.create(ID.unique(),email, password);
```